src/legacy/core_plugins/metrics/public/setup.ts

```diff
@@ -19,17 +19,18 @@
 
 // @ts-ignore
 import { functionsRegistry } from 'plugins/interpreter/registries';
+import { CoreSetup, PluginsServiceSetup } from 'src/core/server';
 import { visualizations } from '../../visualizations/public';
 import { plugin } from './plugin';
 
-const core = {};
+const core = {} as CoreSetup;
 const plugins = {
   data: {
     expressions: {
       functionsRegistry,
     },
   },
   visualizations,
-};
+} as PluginsServiceSetup | any; // TODO: Remove 'any' as soon as PluginsServiceSetup is completed.
```

Comment:
If you write `TsvbSetupPlugins` interface in `plugin.ts`, you can then import it here

Reply:
Added